build.gradle

```diff
@@ -251,32 +253,25 @@ distributions {
 roddyDistTar.dependsOn(jar)
 roddyDistZip.dependsOn(jar)
 
-// IntelliJ IDEA
```

Comment:
Not needed. Also IDEA and Roddy with all the plugins didn't play well recently. Manual project setup is probably the better option.